shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +92,91 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+// Creates and initializes a UIVisualEffectView with a UIBlurEffect. Extracts and returns its
+// gaussianFilter. Returns nil if Apple's API has changed and the filter cannot be extracted.
+- (NSObject*)extractGaussianFilter {
+  NSObject* gaussianFilter = nil;
+
+  for (UIView* view in self.blurEffectView.subviews) {
+    if ([view isKindOfClass:NSClassFromString(@"_UIVisualEffectBackdropView")]) {
+      for (CIFilter* filter in view.layer.filters) {
+        if ([[filter valueForKey:@"name"] isEqual:@"gaussianBlur"]) {
+          if ([[filter valueForKey:@"inputRadius"] isKindOfClass:[NSNumber class]]) {
+            gaussianFilter = filter;
+          }
+          // No need to look at other CIFilters. If the API structure has not changed, the
+          // gaussianBlur filter was succesfully saved. Otherwise, still exit the loop because the
+          // filter cannot be extracted.
+          break;
+        }
+      }
+      // No need to look at other UIViews. If the API structure has not changed, the gaussianBlur
+      // filter was succesfully saved. Otherwise, still exit the loop because the filter cannot
+      // be extracted.
+      break;
+    }
+  }
+
+  return gaussianFilter;
+}
+
+- (BOOL)applyBlurBackdropFilters:(NSArray*)blurRadii {
+  if (!_activeGaussianFilters) {
+    _activeGaussianFilters = [[[NSMutableArray alloc] init] retain];
+
+    _gaussianFilter = [self extractGaussianFilter];
+    [self.blurEffectView release];
```

Comment:
Note: Releases here causes lint unhappy(Which makes sense, there is no guarantee `applyBlurBackdropFilters` is called before `dealoc`. Either make _blurEffectView autorelease, or release at dealloc.